### PR TITLE
Vet clinic database: query and update animals table

### PR DESCRIPTION
**Pull request by techmoves**
in this repo i did this:

- [x] add column called species
- [x] insert newdata in the table

image

image

image
3
image

image

image

image

escape attempte
image

last
image

### CODE REVIEW

**Comment by KDlamini** [comment]
## Status: Invalid 🔴 

### Hi @techmoves 🙋‍♀️,  

While you made a great effort in this project, I can't review your code.

## Invalid Code Review Request
 
You have submitted a project that has a README that is not professional. Here are key points to consider when documenting your README.

- The template logo should be removed. Optionally, the Microverse logo can be replaced with your own project logo if you have one. ✔️
- The only optional section is the FAQ section. If there's no information in the FAQ section then it should be deleted. ✖️ 
- All main headers from the [README template](https://github.com/microverseinc/readme-template) should be present.  ✔️ 
- The `Key Features` and `Future Features` sections should include 1-3 potential features added. ✖️ 
-  Each section description should be removed and replaced with their custom content. The section descriptions are meant to guide you in documenting your README. They are not meant to be part of the final documentation. ✖️ 

For example:
image
image
image

- The License suggestion should be removed. ✖️ 
image

- There should not be any blank sections that do not have content. ✖️ 

Kindly document the `Getting Started` section and sub-headings with the relevant comands.

Kindly document the `Show your support` section.
image
Kindly document the `Acknowledgments` section.
image


- There should not be any broken links in the README file. ✖️
Kindly add the appropriate license file to your project and make sure that the license link works.
- Delete the live demo section if the application is not deployed. If project requirements say to deploy the project, the section should be restored and the link updated. ✖️ 

image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_Invalid Code Review Request **does not count** into the code reviews limit._

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,96 @@
...
+-- Verify that the "species" column has gone back to its state before the transaction
+SELECT * FROM animals;
+
+    -- INSIDE TRANSACTION
+    
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+
+-------------------------------Verify and commit -----
```
Comment (souad988) [inline queries.sql:52]: Great work so far 👍🏼  , but the verification and commit are missing at this point kindly make sure to add them before moving to the next query 

`   SELECT species from animals; -- verify that change was made
   COMMIT;
   SELECT species from animals; -- verify that change persists after commit
`

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,96 @@
...
+
+
+-- Delete all animals born after Jan 1st, 2022.
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+
+-- create a savepoint
```
Comment (souad988) [inline queries.sql:58]: Good job by creating the delete query but the statement to create a save point is missing kindly add it 

``SAVEPOINT SP1;``

File: queries.sql [hunk 3]
```diff
@@ -1,3 +1,96 @@
...
+
+
+-- Update all animals' weight to be their weight multiplied by -1.
+UPDATE animals SET weight_kg = weight_kg * -1;
+
+ROLLBACK;
```
Comment (souad988) [inline queries.sql:64]: the rollback here is needed to go back only this last query without setting a save point the rollback will delete all changes from START TRANSACTION 

``ROLLBACK TO SP1``

File: queries.sql [hunk 4]
```diff
@@ -1,3 +1,96 @@
...
+
+-- Who escapes the most, neutered or not neutered animals?
+SELECT neutered, SUM(escape_attempts) AS escape_count FROM animals GROUP BY neutered
+
+-- What is the minimum and maximum weight of each type of animal?
+SELECT neutered,
+       MIN(weight_kg) AS minimum_weight,
+       MAX(weight_kg) AS maximum_weight
+FROM animals
+GROUP BY neutered;
```
Comment (souad988) [inline queries.sql]: well done here just one adjustement to do is the GROUP BY should be done based on species which defines type of animals not neutered

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @techmoves , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

242722065-db428b0c-8dfc-49c4-a156-42f227e32868


## Highlights
Good PR description
Use of transactions to create scopes for work

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 4 above)
Comment (techmoves) [reply queries.sql]: thanks resolved

File: queries.sql (hunk 3 above)
Comment (techmoves) [reply queries.sql:64]: thanks resolved

File: queries.sql (hunk 2 above)
Comment (techmoves) [reply queries.sql:58]: this it here:
image

File: queries.sql (hunk 4 above)
Comment (techmoves) [reply queries.sql]: image

File: queries.sql (hunk 1 above)
Comment (techmoves) [reply queries.sql:52]: thanks resolved
image

**Review by IngDavidTM — APPROVED** [review]
Hi @techmoves  🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**